src/CouchDB.Client/Query/Selector/SelectorNodesBuilder.cs

```diff
@@ -76,6 +76,10 @@ string GetPropertyName(MemberInfo memberInfo)
                     var op = NewCouchNode(unaryExpr.Operand);
                     return NewUnitaryNode(unaryExpr.NodeType, op);
                 }
+                else if (unaryExpr.NodeType == ExpressionType.Convert && unaryExpr.Operand is MemberExpression)
```

Comment:
Not sure if `unaryExpr.Operand is MemberExpression` is necessary here, but I didn't see a need in other cases

Reply:
It should be ok, thank you.
I'm going to implement this in the new code.

Reply:
Implemented! :)